### DISCUSSION TITLE
Const For Select Query

### DISCUSSION THREAD

**Discussion by dadams39** [microsoftgraph/msgraph-sdk-go#188] [Q&A]
Using options for the `GetWithRequestConfigurationAndResponseHandler(options, handler)` is a great way to reduce the amount of content being pulled from M365. 

I am inquiring to find out if there is a set of const  (e.g.:  "displayName" "id" or "subject") that can be referenced for each model to ensure a `select` is valid?

**Comment by baywet** [comment]
In dotnet and typescript we're suggesting people to use the nameof keyword, and from a quick search [it doesn't seem like Go has an equivalent feature](https://github.com/golang/go/issues/37039). We're not planning to generate those constants at this point. So you can start building your own constants set, even though it's a bit less safe than the nameof solution, Microsoft Graph has a no breaking change policy that ensures the property names on the payload (and consequently filter/select) remain the same in v1.0.
